src/charm.py

```diff
@@ -95,6 +114,36 @@ def _get_pebble_config(self, event: ops.framework.EventBase) -> dict:
             pebble_config["services"]["gunicorn"]["environment"] = pod_env_config
         return pebble_config
 
+    def _get_statsd_pebble_config(self, event: ops.framework.EventBase) -> dict:
+        """Generate statsd exporter pebble config."""
+        pebble_config = {
+            "summary": "gunicorn layer",
+            "description": "gunicorn layer",
```

Comment:
The summary and description should be updated to reflect this is for the statsd layer.